linux/LinuxMDEparser/README.ME

```diff
@@ -0,0 +1,23 @@
+# LinuxMDEparser
+Microsoft Defender Endpoint Log Parser for Linux and MacOS.
+
+Compiled sources (with pyinstaller) can be found on dist folder.
+
+# About
```

Comment:
Please also add a reference to it from linux/README.md (with a one line description what is it for)

Reply:
Regarding this one, can we put it in "Releases" the complied version? The goal is to have the code opensource but for users to use the compiled version and not the python.

I'll provide a Windows and MacOS version